autoexec.cfg

```diff
@@ -133,12 +133,12 @@ snd_headphone_pan_radial_weight "1.0"
 cl_color "1"
 hud_scaling "0.85"
 hud_showtargetid "1"
+cl_hud_playercount_showcount "1" // 1 or 2, Shows remaning alive players in numbers instead of avatars
```

Comment:
Wow, I've been thinking about this forever but haven't taken the time to figure out how. Thanks!